### PR TITLE
Error Messaging Checking for Data Type

### PR DESCRIPTION
**Pull request by mdflynn**
## What is the change? 
This adds a function that will check that the cover image data submitted by the user is a valid image extension.

## Is this a fix or a feature?  
This is a feature

## Where should the reviewer start? 
From lines 158 - 183 on the main.js file

## How should this be tested?
Open index.html in your terminal. Navigate to the create your cover section, try to submit data without all fields field. Try to submit a non-image link in cover input to see error messaging.

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -152,6 +155,32 @@ function dataValidate(data) { 
   } 
 } 
 
+
```
Comment (coopterrones) [inline src/main.js:158]: looks great! data validation on point!

File: src/main.js [hunk 2]
```diff
@@ -152,6 +155,32 @@ function dataValidate(data) {
...
+    if (inputFields[i].value == '') {
+      makeABookButton.disabled = true;
+      return false;
+    } else {
+      makeABookButton.disabled = false;
+    }
+  }
+}
+
+function errorMessage() {
```
Comment (coopterrones) [inline src/main.js:170]: error message alerts great when jpg

**Review by coopterrones — APPROVED** [review]

File: src/main.js (hunk 2 above)
Comment (Ish451) [reply src/main.js:170]: looks a lot cleaner from our version

Comment (Ish451) [reply src/main.js:170]: *1st version

**Pull request merged**